src/fastgltf.cpp

```diff
@@ -419,6 +439,42 @@ std::pair<fg::Error, fg::DataSource> fg::glTF::loadFileFromUri(URI& uri) const n
     return std::make_pair(Error::None, std::move(vectorSource));
 }
 
+#if defined(__ANDROID__)
+std::pair<fg::Error, fg::DataSource> fg::glTF::loadFileFromApk(const std::filesystem::path& filepath) const noexcept {
+    const auto fileString = filepath.string();
+    auto assetDeleter = [](AAsset* file) { AAsset_close(file); };
+    auto file = std::unique_ptr<AAsset, decltype(assetDeleter)>(AAssetManager_open(androidAssetManager, fileString.c_str(), AASSET_MODE_BUFFER), assetDeleter);
```

Comment:
Perhaps these two lines could become a macro or something as they're duplicated from `loadFromAndroidAsset` and are 100% identical, and long.